airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/parquet/ParquetSerializedBuffer.java

```diff
@@ -86,6 +92,56 @@ public ParquetSerializedBuffer(final S3DestinationConfig config,
     lastByteCount = 0L;
   }
 
+  private void installNativeLzoLibraries() throws IOException {
+    final String osName = System.getProperty("os.name").replace(' ', '_');
+    final String currentDir = System.getProperty("user.dir");
+    final String architecture = resolveArchitecture();
+
+    LOGGER.info("OS {} architecture {}: ", osName, architecture);
+    Runtime runtime = Runtime.getRuntime();
+    if (osName.equals("Linux")) {
+      if (architecture.equals("amd64-64")) {
+        try {
+          runProcess(currentDir, runtime, "/bin/sh", "-c", "apt-get update");
+          runProcess(currentDir, runtime, "/bin/sh", "-c", "apt-get install lzop liblzo2-2 liblzo2-dev -y");
```

Comment:
Why such very "dynamic" way of installing this software ?
I think you can install it in advance on stage of Docker build
What do you think?

Reply:
I agree. these should be installed on the connector's docker image

Reply:
> I agree. these should be installed on the connector's docker image

@girarda @grubberr  moved this logic to the Dockerfile